package.json

```diff
@@ -1209,7 +1215,7 @@
     "vscode:prepublish": "webpack --env.production --config-name webview && webpack --env.production --config-name extension && npx webpack-cli --config browser.webpack.config --env.production --config-name extension --mode none",
```

Comment:
Any script that has `--config-name webview` needs to be updated to include the appropriate `--config-name` for the activity bar webview.

Reply:
and `--config-name webview` needs to be changed to `--config-name webviewIndex.js`

Reply:
Thanks for catching this!